pkg/kubectl/cmd/helpers.go

```diff
@@ -171,3 +173,17 @@ func ReadConfigDataFromLocation(location string) ([]byte, error) {
 		return data, nil
 	}
 }
+
+func PrintObject(cmd *cobra.Command, obj runtime.Object, f *Factory, out io.Writer) {
+	mapper, _ := f.Object(cmd)
+	defaultVersion, kind, err := mapper.VersionAndKindForResource("replicationcontrollers")
```

Comment:
Is there not an easy way to make this work for any kind? Could it be extracted from the object? Or perhaps just make kind a parameter to the function? 

This is how I did it in the apiserver:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/apiserver/apiserver.go#L109

Maybe @smarterclayton knows a better way.